packages/@react-spectrum/datepicker/docs/DatePicker.mdx

```diff
@@ -0,0 +1,312 @@
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/datepicker';
+import i18nDocs from 'docs:@internationalized/date';
+import {HeaderInfo, PropTable, PageDescription, TypeLink} from '@react-spectrum/docs';
+import packageData from '@react-spectrum/datepicker/package.json';
+
+```jsx import
+import {DatePicker} from '@react-spectrum/datepicker';
+import {Flex} from '@react-spectrum/layout';
+```
+
+---
+category: Pickers
+keywords: []
+after_version: 3.0.0
+---
+
+# DatePicker
+
+<PageDescription>{docs.exports.DatePicker.description}</PageDescription>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['DatePicker']}
+  sourceData={[]} />
+
+## Example
+
+```tsx example
+<DatePicker label="Event date" />
+```
+
+## Value
+
+A `DatePicker` displays a placeholder date by default. An initial, uncontrolled value can be provided to the `DatePicker` using the `defaultValue` prop. Alternatively, a controlled value can be provided using the `value` prop.
+
+Date values are provided using objects in the `@internationalized/date` package. This library handles correct international date manipulation across calendars, time zones, and other localization concerns. `DatePicker` supports values of the following types:
+
+* <TypeLink links={i18nDocs.links} type={i18nDocs.exports.CalendarDate} /> – a date without any time components. May be parsed from a string representation using the <TypeLink links={i18nDocs.links} type={i18nDocs.exports.parseDate} /> function. Use this type to represent dates where the time is not important, such as a birthday or an all day calendar event.
+* <TypeLink links={i18nDocs.links} type={i18nDocs.exports.CalendarDateTime} /> – a date with a time, but not in any specific time zone. May be parsed from a string representation using the <TypeLink links={i18nDocs.links} type={i18nDocs.exports.parseDateTime} /> function. Use this type to represent times that occur at the same local time regardless of the time zone, such as the time of New Years Eve fireworks. Most times are better stored as a `ZonedDateTime`.
+* <TypeLink links={i18nDocs.links} type={i18nDocs.exports.ZonedDateTime} /> – a date with a time in a specific time zone. May be parsed from a string representation using the <TypeLink links={i18nDocs.links} type={i18nDocs.exports.parseZonedDateTime} />, <TypeLink links={i18nDocs.links} type={i18nDocs.exports.parseAbsolute} />, or <TypeLink links={i18nDocs.links} type={i18nDocs.exports.parseAbsoluteToLocal} /> functions. Use this type to represent an exact moment in time at a particular location on Earth.
+
+```tsx example
+import {parseDate} from '@internationalized/date';
+
+function Example() {
+  let [value, setValue] = React.useState(parseDate('2020-02-03'));
+
+  return (
+    <Flex gap="size-150" wrap>
+      <DatePicker
+        label="Date (uncontrolled)"
+        defaultValue={parseDate('2020-02-03')} />
```

Comment:
haha! so I don't get to use my birthday? 😉

Reply:
Is it still not clear though because the year is first?

Reply:
lol yes it is once I look at the code but just seeing the component rendering and reading the descriptions about locales makes me wonder what format I am looking at. It would be nicer to see that without having to check the code.
and for sure, someone will steal your identity with your partial bday :)